lighthouse-core/audits/mainthread-work-breakdown.js

```diff
@@ -52,9 +52,9 @@ class MainThreadWorkBreakdown extends Audit {
    */
   static get defaultOptions() {
     return {
-      // see https://www.desmos.com/calculator/s2eqcifkum
-      scorePODR: 1500,
-      scoreMedian: 4000,
+      // see https://www.desmos.com/calculator/vhglu1x8zv
+      p10: 2017,
```

Comment:
feels like this one could be adjusted to 2000 or just left as-is :)

Reply:
> feels like this one could be adjusted to 2000 or just left as-is :)

I was just using the exact (nearest integer) p10 values given the existing curve for all these. I'll take another pass and see which ones can be rounded more without substantial effect. I just didn't want to get into examining each curve, whether it was worth rounding by some amount X at the expense of score drifts of Y when we're not actually changing anything fundamental this time on scoring. Better to save it to coincide with the next round of real score adjustments :)

Reply:
Yeah that's fine, I think this is mostly just exposing that our 90 cutoffs were super weird before lol